packaging/installer/kickstart-static64.sh

```diff
@@ -1,5 +1,5 @@
-#!/usr/bin/env sh
-#
+#!/bin/sh
```

Comment:
Same as for `netdata-installer.sh`

```suggestion
#!/usr/bin/env sh
```